tests/unit/backend/layers/api/test_curation_api.py

```diff
@@ -1567,7 +1597,39 @@ def test_get_datasets_nonexistent_collection_4xx(self):
             response = self.app.get(test_url, headers=headers)
             self.assertEqual(403, response.status_code)
 
-    def test_get_all_datasets_200(self):
+    @patch("backend.common.corpora_config.CorporaConfig.__getattr__")
+    def test_get_dataset_no_assets(self, mock_config: Mock):
+        def mock_config_fn(name):
```

Comment:
suggestion: Reuse `mock_config_fn`? It could then be used in the patch decorator and parameterized as needed.

```
@patch("backend.common.corpora_config.CorporaConfig.__getattr__", side_effect=mock_config_fn())
```

Also I think this could be simplified to use a dict rather than a bunch of ifs and returns.
```
def mock_config_fn(name):
   mocked_attr = { 
       "curator_role_arn":"test_role_arn",
       ""submission_bucket": "cellxgene-dataset-submissions-test",
      ....
}
      return mocked_attr[name]
```

The mocked_attr dict could even be defined outside of the function to avoid invoking it every time.